backend/api/tests/test_models.py

```
@@ -0,0 +1,222 @@
+        assert profile.occupation == "Engineer"
+        assert profile.content_preferences == ["Technology", "Science"]
+        assert not profile.onboarding_completed
+    
+    def test_viewer_profile_str(self):
+        # Test the string representation
+        user = User.objects.create_user(
+            email="viewer1@example.com",
+            firebase_uid="vieweruid1"
+        )
+        profile = ViewerProfile.objects.create(user=user)
+        assert str(profile) == f"Profile for viewer1@example.com"
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)